views/tasks/active-pscs.html

```diff
@@ -0,0 +1,81 @@
+{% extends "layout.html" %}
+{% block pageTitle %}
+    {% if errorList.length > 0 %}
+        Error:
+    {% endif %}
+    Review the PSCs
+{% endblock %}
+{% block backLink %}
+  {{ govukBackLink({
+    text: "Back",
+    href: backLinkUrl
+  }) }}
+{% endblock %}
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-three-quarters">
+            <form action="" method="post">
```

Comment:
i think we can remove the empty action="" from the form and it should still work the same

Reply:
Removed